.github/workflows/cypress.yml

```diff
@@ -33,36 +42,51 @@ jobs:
       - name: Install bootstrapper dependencies
         run: pipenv install --dev --deploy
       - run: pipenv run cookiecutter . --no-input -f
+      - uses: actions/upload-artifact@v3
+        with:
+          name: my_project
+          path: my_project/
+          retention-days: 1
+    outputs:
+      BASE_URL: ${{ env.BASE_URL }}
```

Comment:
`upload-artifacts` allows us to do all the work we needed to do for cookiecutter only once. The later workflows can then do things like run inside a docker container and not worry about pip missing or not having permissions to install things (problem I ran into)